fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -5036,7 +5035,8 @@ TEST_CASE("noSim/ShardedRocksDB/RocksDBSstFileWriter") {
 		ASSERT(status.ok());
 		ASSERT(value == targetValue.toString());
 	}
-	delete db;
+	auto s = db->Close();
+	ASSERT(s.ok());
```

Comment:
From examples https://github.com/facebook/rocksdb/wiki/basic-operations, it seems `delete db` is still needed after calling `close`.

Reply:
good catch!